src/platform/wii/main.c

```diff
@@ -892,6 +899,20 @@ uint16_t _pollGameInput(struct mGUIRunner* runner) {
 			keys |= 1 << angles;
 		}
 	}
+	int a_l = PAD_TriggerL(0);
+	int a_r = PAD_TriggerR(0);
+	if (a_l > 0x50) {
+		keys |= 1 << GBA_KEY_L;
```

Comment:
Don't hardcode these. They should be bound to the same buttons as L and R, or else it would get very confusing.